scripts/create_dataset.py

```diff
@@ -1,69 +1,161 @@
+
+
+def create_column_and_relationships(input_relationships,
+                                    column_name,
+                                    column_datatype,
+                                    column_array_of,
+                                    input_table_name,
+                                    input_tables):
+    """
+    Creates a column and relationships, keeping ony the name, data type and column field from the input column.
+    :param input_relationships: the list of relationships from an input JSON column
+    :param column_name: the name of the input JSON column
+    :param column_datatype: the primitive type enforced for values of the input JSON column
+    :param column_array_of: indicates if the column values should be in array
+    :param input_table_name: the name of input JSON table that has the input_column
+    :param input_tables: the input tables that have been added to a dict and used to validate the created relationships
+    """
+    relationships = [
+        {
+            "name": "{}_{}_to_{}_{}".format(input_table_name,
+                                            column_name,
+                                            input_relationship["table_name"],
+                                            input_relationship["column_name"]),
+            "from": {"table": input_table_name, "column": column_name, "cardinality": "one"},
+            "to": {
+                    "table": input_relationship["table_name"],
+                    "column": input_relationship["column_name"],
+                    "cardinality": "many" if column_array_of is True else "one"
+                  }
+        }
+        if validate_table_and_column(
+               table_name=input_table_name,
+               column_name=column_name,
+               input_tables=input_tables) and validate_table_and_column(table_name=input_relationship["table_name"],
+                                                                        column_name=input_relationship["column_name"],
+                                                                        input_tables=input_tables)
+        else []
+        for input_relationship in input_relationships] if input_relationships else []
+
+    column = {
+        "name": column_name,
+        "datatype": "string" if column_datatype == "link" else column_datatype,
+        "array_of": column_array_of is True
+    }
+
+    return column, relationships
+
+
+def validate_table_and_column(table_name, column_name, input_tables):
+    """
+    Validates a relationship checking the the given table and column exist in input tables.
+    :param table_name: the name of table containing has the input_column that will be validated against the input tables
+    :param column_name: the name of the input_column that will be validated against the input tables
+    :param input_tables: the input tables that have been added to a dict and used to validate the created relationships
+    """
+    if True in [
```

Comment:
`any` is the same as `True in` when the list elements are booleans, and it's more idiomatic.